source/apiVolontaria/volunteer/views.py

```diff
@@ -309,6 +312,55 @@ def get_queryset(self):
             queryset = queryset.filter(user__username=username)
         return queryset
 
+    def post(self, request, *args, **kwargs):
+        response = self.create(request, *args, **kwargs)
+
+        participation = models.Participation.objects.get(
+            user=self.request.user,
+            event__id=request.data["event_id"],
+        )
+
+        # If it's a success
+        if response.status_code == status.HTTP_201_CREATED:
+            # If we have a configured service of email
+            if settings.CONSTANT['EMAIL_SERVICE'] is True:
+                # If we want to confirm participation creation with an email
+                if settings.CONSTANT['SEND_EMAIL_CONFIRM_CREATE_PARTICIPATION']:
```

Comment:
- Could merge the 2 if statement to reduce the indentation level
- Coud check for `is True` to be more verbose (and to be consistent with 
```
if settings.CONSTANT['EMAIL_SERVICE'] is True:
```
)